tests/end2end/playwright/attribute-table.spec.js

```diff
@@ -56,19 +58,147 @@ test.describe('Attribute table data restricted to map extent', () => {
+                {layer:'single_wms_lines_group_as_layer', onPage:0},
+                {layer:'single_wms_polygons', onPage:1},
+            ]
+        },
+        {
+            login:'user_in_group_a',
+            enabled_groups: ['group_a','group_b'],
+            expected: [
+                {layer:'single_wms_points', onPage:1},
+                {layer:'single_wms_points_group', onPage:1},
+                {layer:'single_wms_lines_group_as_layer', onPage:0},
+                {layer:'single_wms_polygons', onPage:1},
+            ]
+        },
+        {
+            login:'user_in_group_b',
+            enabled_groups: ['admins'],
+            expected: [
+                {layer:'single_wms_points', onPage:0},
+                {layer:'single_wms_points_group', onPage:0},
+                {layer:'single_wms_lines_group_as_layer', onPage:0},
+                {layer:'single_wms_polygons', onPage:1},
+            ]
+        },
+        {
+            login:'publisher',
+            enabled_groups: ['group_b','group_a','admins'],
+            expected: [
+                {layer:'single_wms_points', onPage:0},
+                {layer:'single_wms_points_group', onPage:0},
+                {layer:'single_wms_lines_group_as_layer', onPage:0},
+                {layer:'single_wms_polygons', onPage:0},
+            ]
+        },
+    ].forEach(({login, enabled_groups, expected}, c) => {
+        test(`#${c} Layer export with ${login} user logged in`, {
+            tag: '@write',
+        }, async ({browser}) => {
+            // open admin page to set export permissions
+            const adminContext = await browser.newContext({ storageState: getAuthStorageStatePath('admin') });
+            const page = await adminContext.newPage();
+            const adminPage = new AdminPage(page);
 
-        await page.getByRole('button', { name: 'Export' }).click();
-        await page.getByRole('button', { name: 'GeoJSON' }).click();
+            await page.goto('admin.php');
+            // open maps management page
+            await adminPage.openPage('Maps management');
 
-        getFeatureRequestContains(await getFeatureRequestPromise);
-    });
+            // set layer export permissions
+            await adminPage.modifyRepository('testsrepository');
+            await adminPage.uncheckAllExportPermission();
+            await adminPage.setLayerExportPermission(enabled_groups);
+            await adminPage.page.getByRole('button', { name: 'Save' }).click();
+
+            // login with specific user
+            let userContext;
+            if (login !== '__anonymous') {
+                userContext = await browser.newContext({storageState: getAuthStorageStatePath(login)});
+            } else {
+                userContext = await browser.newContext();
+            }
+            const userPage = await userContext.newPage();
+
+            // go to project page
+            const project = new ProjectPage(userPage, 'enable_export');
```

Comment:
```suggestion
            const project = new ProjectPage(userPage, 'layer_export_acl');
```

Do you mind renaming the project ? Sorry nitpicking, in the source code, the word `ACL` is usually used. So it would be easier to find projects, E2E tests with this word I think later.

Similar to https://github.com/3liz/lizmap-web-client/blob/master/tests/qgis-projects/tests/project_acl.qgs

Or looking at keywords projects in the list https://sandbox.lizmap.com/lizmap_3_8/